services/web-app/pages/about-carbon/how-carbon-works.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: How Carbon works
+description:
+  Carbon is IBM’s design system for products and digital experiences. With the IBM Design Language
+  as its foundation, the system consists of working code, design tools and resources, human
+  interface guidelines, and a vibrant community of contributors.
+---
+
+import { section } from './how-carbon-works.module.scss'
+
+<PageDescription>
+  Carbon is IBM’s design system for products and digital experiences. With the IBM Design Language
+  as its foundation, the system consists of working code, design tools and resources, human
+  interface guidelines, and a vibrant community of contributors.
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>The Carbon ecosystem</AnchorLink>
+  <AnchorLink>Creating consistency </AnchorLink>
+  <AnchorLink>The people behind Carbon</AnchorLink>
+</AnchorLinks>
+
+## The Carbon ecosystem
+
+Carbon is s system of systems. At the epicenter lies **Carbon’s universal system**, with assets that
+can be used as a base for the widest variety of situations. These components, patterns, elements,
+and code are also open source and can easily be extended or adapted.
+
+Additionally, teams throughout IBM have been able to abstract components out and create their own
+**product and digital systems**. Examples include
+[Carbon for IBM.com](https://www.ibm.com/standards/carbon/) and
+[Carbon for IBM Products](https://pages.github.ibm.com/cdai-design/pal/).
+
+![Carbon ecosystem](/about-carbon/ecosystem.png)
+
+If you’re ready to get started with Carbon, check out our guidance for
+[designers](/designing/get-started) and [developers](/developing/get-started).
+
+Today, we are platforming this system of systems, giving makers both **global visibility and local
+lenses**. Our goal is to help makers find assets that comply with platform requirements, are
+convenient to implement, and are consistent with design patterns across the company.
+
+For the first time, makers will be able to find and re-use assets across all IBM teams, regardless
+of who maintains them.
+
+Additionally, makers can quickly find assets that are compliant with their own product or team
+through local lenses.
+
+Head over to the [component catalog](/components) to see the many assets shared between product and
+digital teams. Progress on the new platform can be found in our
+[roadmap](/about-carbon/platform-roadmap).
+
+## Creating consistency
+
+The Carbon Design System now follows a schema to document each resource in a standardized way. This
```

Comment:
Add a link to the GH schema doc under 'Creating Consistency': The Carbon Design System now follows a schema (link)